pkg/session/bench_test.go

```diff
@@ -1949,6 +1950,8 @@ var batchNum = 100
 var batchSize = 100
 
 func BenchmarkPipelinedSimpleInsert(b *testing.B) {
+	failpoint.Enable("tikvclient/pipelinedSkipResolveLock", "return")
```

Comment:
Does it mean the it's tested without commit the prewrite locks?

Reply:
Yes. Hopefully it will make the test more stable.